### PR TITLE
adding Opster to partners list

### PR DESCRIPTION
**Pull request by segalziv**
### Description
[Describe what this change achieves]
Adding opster to the partners list. see publication: https://opster.com/blogs/announcing-opensearch-support/
 
### Issues Resolved
[List any issues this PR will resolve]

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _partners/opster.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: Opster
+logo: 'https://opster.com/wp-content/uploads/2021/07/Opster-logo-for-opensearch.png'
```
Comment (stockholmux) [inline _partners/opster.md:3]: I'm going to approve this, but do you mind if I grab a version of your logo from your website without a white background and store it as part of the website repo (hot linking too fragile for my tastes).

Screen Shot 2021-07-06 at 3 35 07 PM

**Review by stockholmux — APPROVED** [review]
See note on the logo

File: _partners/opster.md (hunk 1 above)
Comment (segalziv) [reply _partners/opster.md:3]: thanks @stockholmux , yes sure - please feel free to use a proper Opster logo that fit the website. you can download it and store on the website repo too. thanks

Comment (segalziv) [reply _partners/opster.md:3]: Opster-logo-for-opensearch

@stockholmux I added the logo image without background here for your convenience - let me know if you need anything.

**Pull request merged**